UndertaleModTool/SampleScripts/SearchLimited.csx

```diff
@@ -56,7 +56,7 @@ if (regex_check)
         keywordRegex = new(keyword, RegexOptions.Compiled | RegexOptions.IgnoreCase);
 }
 
-string[] searchNamesList = searchNames.Split('\n', StringSplitOptions.RemoveEmptyEntries);
+string[] searchNamesList = searchNames.Split(new[] { '\n', '\v' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
This case should get handled in SimpleTextInput rather than in the script itself.